yarn-project/aztec.js/src/account_manager/account_manager.ts

```diff
@@ -248,31 +248,6 @@ export class AccountManager {
     return new DeployAccountSentTx(this.pxe, sentTx, this.getWallet());
   }
 
-  /**
-   * Estimates the gas needed to deploy the account contract that backs this account.
-   * This method is here to ensure that the fee payment method is correctly set up in case
-   * the account contract needs to pay for its own deployment.
-   * @param opts - Fee options to be used for the deployment.
-   * @returns The gas estimations for the account contract deployment and initialization.
-   */
-  public async estimateDeploymentGas(
-    opts?: DeployAccountOptions,
-  ): Promise<Pick<GasSettings, 'gasLimits' | 'teardownGasLimits'>> {
-    const deployMethod = await this.#getDeployMethod(opts?.deployWallet);
-    const fee =
-      !opts?.deployWallet && opts?.fee
-        ? { ...opts.fee, paymentMethod: await this.#getSelfPaymentMethod(opts.fee.paymentMethod) }
-        : opts?.fee;
-    return deployMethod.estimateGas({
-      contractAddressSalt: new Fr(this.salt),
-      skipClassRegistration: opts?.skipClassRegistration ?? true,
-      skipPublicDeployment: opts?.skipPublicDeployment ?? true,
-      skipInitialization: opts?.skipInitialization ?? false,
-      universalDeploy: true,
-      fee,
-    });
-  }
-
```

Comment:
?

Reply:
This was unnecessary. I added comments where this was used to clarify.